test/cypress/integration/request-to-go-live/organisation_name_spec.js

```diff
@@ -55,21 +37,33 @@ describe('Request to go live: organisation name page', () => {
     }]
   }
 
-  const setupStubs = (serviceRole) => {
-    cy.task('setupStubs', simpleStub(serviceRole))
+  const serviceRoleBeforePatchRequests = utils.buildServiceRoleForMerchantDetailsField({ name: organisationName })
+  const serviceRoleAfterPatchRequests = utils.buildServiceRoleForGoLiveStage('ENTERED_ORGANISATION_NAME')
+  function getUserSuccess (serviceRolesBefore = [serviceRoleBeforePatchRequests], serviceRolesAfter = serviceRoleAfterPatchRequests) {
```

Comment:
I think using the default parameters here makes it slightly confusing, I think it would be better to create the service roles per test and always pass them in.

Reply:
Is this function also a candidate to go into the utils, as we use it in other _specs?

Reply:
Good thinking! I've added this to the utils folder so it can be used for future refactoring. I guess we could start refactoring once we're done, so we're not all working on the same common modules at once.